portfolio/src/main/webapp/index.html

```diff
@@ -9,9 +9,12 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <img src="23a66875-c185-4420-a530-a703f21b15d2 (1).jpg" alt="Murori">
```

Comment:
This is not a good image file name, but I think it should be still considered valid and display the image correctly on your deployed AppEngine webapp, but I'm not seeing this image in your rendered portfolio web page. I suspect that the blank space in the file name and the parentheses might be not well received by the AppEngine?

The best practice is to put all images in a separate directory (e.g., `portfolio/src/main/webapp/images/foo.jpg`), and link to it as `<img src="images/foo.jpg" />`. I'm seeing that you did created `images` directory, but under `WEB-INF`, not under `webapp`. If you move your `images` directory into the `webapp` directory, and modify your link here to `<img src="images/IMG-xxx.jpg"/>`, then I think it should display the image correctly in the rendered page.